ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -226,6 +226,64 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53-registry-s3-cloudfront
+- as: aws-ipi-clusterapi-mixed-workers-tp-amd-f28-destructive
+  cron: 52 17 15 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
Is there a dedicate `FEATURE_GATES` (plus `FEATURE_SET: CustomNoUpgrade`) to control the api you want to enable, but not have to enable all the TP features.

Reply:
I remember there is an email about prompting a TP feature to GA, it is encouraged to use `FEATURE_GATES` + `FEATURE_SET: CustomNoUpgrade` to run testing, and monitoring its pass ratio using sippy tools. As a TP feature, I think the most important things is to ensure the feature is working in a stable way as the expectation without other TP feature noise. About combination with other features, I think that is in the scope of integration testing as a regular GA feature.

Reply:
@jianlinliu we have a card moving the feature from `CustomNoUpgrade` to `TechPreviewNoUpgrade` https://issues.redhat.com/browse/OCPCLOUD-2914 then we only add `CustomNoUpgrade` jobs to run testing is ok?

Reply:
If that is on purpose from your side, I am okay with it.

Reply:
@jianlinliu do you happen to have the email at hand? I thought the tests to be displayed in sippy featuregate depend on the e2e tests having the `[OCPFeatureGate:<FeatureGateName>]` label. maybe i'm misunderstanding

Reply:
Email name  - `New feature gates page in Sippy`:
```
Tests must contain either [OCPFeatureGate:<FeatureGateName>] or the standard upstream [FeatureGate:<FeatureGateName>].
```

`FEATURE_GATES` in the prow ci will be applied to install-config.yaml as `featureGates`, I am not sure if the sippy can capture the installation, maybe I am wrong either.